kubernetes/test-infra/aks-new/main.tf

```diff
@@ -0,0 +1,65 @@
+provider "azurerm" {
+  features {}
```

Comment:
when attempting to use `export ARM_SUBSCRIPTION_ID` it doesn't seem to apply the id correctly. Using the attribute `subscription_id` within the provider block solves this issue.